pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/ingress_reconciler.go

```diff
@@ -78,9 +78,43 @@ func (ir *IngressReconciler) Reconcile(ctx context.Context, isvc *v1beta1.Infere
 	serviceHost := getServiceHost(isvc)
 	serviceUrl := getServiceUrl(isvc, ir.ingressConfig)
 	disableIstioVirtualHost := ir.ingressConfig.DisableIstioVirtualHost
+
+	// Check if the isvc is stopped
```

Comment:
The current comment might be misleading. To improve clarity, consider changing it to:

```suggestion
	// If the "serving.kserve.io/stop" annotation is present in isvc, follow the stop-handling logic
```